docs/content/toc.textile

```diff
@@ -64,6 +64,7 @@ h2. Operations
 * "Different Hadoop Versions":../operations/other-hadoop.html
 * "Multitenancy Considerations":../operations/multitenancy.html
 * "Performance FAQ":../operations/performance-faq.html
+* "Insert Segments":../operations/insert-segment-to-db.html
```

Comment:
i don't like this TOC heading and this getting its own section

I prefer we rework the libraries section for Druid and for that page to link to the doc here